dm/syncer/syncer.go

```diff
@@ -1369,6 +1349,26 @@ func (s *Syncer) syncDDL(queueBucket string, db *dbconn.DBConn, ddlJobChan chan
 			}
 		}
 
+		// set timezone
+		if ddlJob.timezone != "" {
+			s.timezoneLastTime = ddlJob.timezone
+			setTimezoneSQL := fmt.Sprintf("SET SESSION TIME_ZONE = '%s'", ddlJob.timezone)
+			ddlJob.ddls = append([]string{setTimezoneSQL}, ddlJob.ddls...)
+			setTimezoneSQLDefault := "SET SESSION TIME_ZONE = DEFAULT"
+			ddlJob.ddls = append(ddlJob.ddls, setTimezoneSQLDefault)
+		} else if s.timezoneLastTime != "" {
+			// use last time's time zone
+			setTimezoneSQL := fmt.Sprintf("SET SESSION TIME_ZONE = '%s'", s.timezoneLastTime)
+			ddlJob.ddls = append([]string{setTimezoneSQL}, ddlJob.ddls...)
+			setTimezoneSQLDefault := "SET SESSION TIME_ZONE = DEFAULT"
+			ddlJob.ddls = append(ddlJob.ddls, setTimezoneSQLDefault)
+		}
+		// set timestamp
+		setTimestampSQL := fmt.Sprintf("SET TIMESTAMP = %d", ddlJob.timestamp)
+		ddlJob.ddls = append([]string{setTimestampSQL}, ddlJob.ddls...)
+		setTimestampSQLDefault := "SET TIMESTAMP = DEFAULT"
+		ddlJob.ddls = append(ddlJob.ddls, setTimestampSQLDefault)
+
```

Comment:
seems these line can be further moved into `if !ignore {}` in line 1378

Reply:
but when `ignore` is true, these lines would not be executed, and the timestamp and timezone would not be set back to default, I'm not sure whether this would cause some bugs